lib/test.gi

```diff
@@ -536,7 +543,7 @@ end);
 ## 
 ##    testOptions := rec()   : Options to pass on to Test
 ##    earlyStop := false     : Stop once one test fails
-##    showProgress := true   : Show progress
+##    showProgress := "some" : Show some progress
```

Comment:
I would make the comment more clear, e.g. `Show some progress (the number of the current line of the test being processed)` or a bit shorter - but more informative then just "some progress".